src/components/map/MapDirective.js

```diff
@@ -125,6 +124,60 @@ goog.require('ga_permalink');
                 gaOffline.hideExtent();
               }
             });
+
+            var savedTimeStr = {};
+            scope.$on('gaTimeChange', function(evt, time, oldTime) {
+              var switchTimeActive = (!oldTime && time);
+              var switchTimeDeactive = (oldTime && !time);
+              var olLayers = scope.map.getLayers().getArray();
+              var singleModif = false;
+
+              // Detection the time change has been triggered by a layer's
+              // 'propertychange' event.
```

Comment:
So, when changing the time of a layer in the layerManager and this change results in every layer having the same time, then `singleModif` will be true here, right?

I just can't see how the layermanager triggers a gaTimeChange event. I also don't see where it calls gaTime.set.